### PR TITLE
Added support to more languages #27 

### LINKED ISSUE #27: Add support to more languages
Add support for at least two new languages in one PR.
Eg: Go, JS etc.

### PR DESCRIPTION
**Pull request by prithaupadhyay**
Fixes #27

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
Hi! Thanks for PR.
Make sure you have checked points in PR Template correctly.

(Powered by [GitMate.io](https://gitmate.io))

File: tc_generator/logic.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+             long n,x,res;
+             n = long.Parse(Console.ReadLine());
+             x= (n*(n+1))/2;
+             res=x*x;
+             Console.WriteLine(res%1000000007 + "\n");
+             t--;
+         }
+      }
+   }
+}
```
Comment (aashutoshrathi) [inline tc_generator/logic.cs]: Newline at the end, also make sure you indent it properly.

File: tc_generator/tc_gen.py [hunk 2]
```diff
@@ -53,6 +53,12 @@
         'link': 'https://www.python.org/downloads/',
         'compile': ''
     },
+    {
+        'req': 'C#',
+        'command': 'logic',
```
Comment (aashutoshrathi) [inline tc_generator/tc_gen.py]: Is command just `logic`?

Comment (prithaupadhyay) [reply tc_generator/tc_gen.py]: I have made the necessary changes. Please do have a look.

**Comment by prithaupadhyay** [comment]
@aashutoshrathi  I have added the Golang code for logic. Please do have a look.

File: tc_generator/tc_gen.py [hunk 3]
```diff
@@ -87,8 +99,8 @@ def zip_them(test_files, choice):
 
 def main():
     choice = int(INPUT(
-        "Enter your choice of language\n1 for C\n2 for C++\n3 for Java\n4 for Python\n"))
-    if choice not in (1, 2, 3, 4):
+        "Enter your choice of language\n1 for C\n2 for C++\n3 for Java\n4 for Python\n5 for C#\n6 for Golang"))
+    if choice not in (1, 2, 3, 4, 5, 6):
```
Comment (aashutoshrathi) [inline tc_generator/tc_gen.py]: Can you check if we can write in a better way like `in range(1:6)`

Comment (prithaupadhyay) [reply tc_generator/tc_gen.py]: On it.

**Pull request merged**